packages/blazor-workspace/Examples/Demo.Shared/Pages/ComponentsDemo.razor.cs

```diff
@@ -313,6 +315,11 @@ public void RemoveDiesFromRadius(int numberOfDies)
     {
         UpdateDiesTable(DiesTable.Length - (int)(numberOfDies * numberOfDies * Math.PI));
     }
+
+    public void UpdateUserMessages(ChatInputSendEventArgs e)
+    {
+        _userMessages.Add(e.Text);
```

Comment:
For posterity / other reviewers: the Angular app calls `resetInput()` here but we're not doing this from Blazor because Meyer is planning to remove that method and have the component do that automatically when the send button is pressed.